src/components/swap/TradePrice.tsx

```diff
@@ -33,7 +33,14 @@ export default function TradePrice({ price }: TradePriceProps) {
   const { baseCurrency, quoteCurrency } = price
   const { data: usdPrice } = useUSDPrice(tryParseCurrencyAmount('1', showInverted ? baseCurrency : quoteCurrency))
 
-  const formattedPrice = formatPrice(showInverted ? price : price.invert(), NumberType.TokenTx)
+  let formattedPrice: string
```

Comment:
nit: if you put this in a helper function you can continue to use a `const` (which is generally less error-prone):

```
const formattedPrice = useMemo(() => {
  try {
    return formatPrice(showInverted ? price : price.invert(), NumberType.TokenTx)
  } catch {
    return '0'
  }
}, [price, showInverted])
```